src/quo2/components/profile/profile_card/style.cljs

```diff
@@ -0,0 +1,33 @@
+(ns quo2.components.profile.profile-card.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def card-container
+  {:flex-direction   :column
+   :padding          12
+   :flex             1
+   :border-radius    16
+   :background-color (colors/custom-color :turquoise 50 40)})
```

Comment:
Color (`:turquoise`) shouldn't be hard coded, it should be passed as a parameter. We will allow the user to customize their profile and choose a theme color.

Reply:
Thank you! Already did the change!